.pipelines/Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM ubuntu:16.04
+FROM ubuntu:20.04
 RUN apt-get update && apt-get install -y software-properties-common sudo wget apt-transport-https curl lsb-release gnupg jq
 RUN wget -q https://packages.microsoft.com/config/ubuntu/16.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
 RUN sudo dpkg -i packages-microsoft-prod.deb
 RUN add-apt-repository ppa:longsleep/golang-backports 
 RUN curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
 RUN curl -fsSL https://get.docker.com -o get-docker.sh && sh get-docker.sh
 RUN apt-get install -y git golang-go=2:1.13~1longsleep1+xenial iptables ipset iproute2 ebtables python-pip gcc zip dotnet-sdk-2.2 azure-cli
```

Comment:
let's update this to golang 1.14 while we're at it

Reply:
bumped